src/librustc_dep_graph/lib.rs

```diff
@@ -0,0 +1,44 @@
+#![feature(core_intrinsics)]
```

Comment:
Hmm; what is this used by?

Reply:
This is the `unlikely` macro, which is used in `try_mark_previous_green`.